### DISCUSSION TITLE
The existing GitHub project was deleted due to a new connection.

### DISCUSSION THREAD

**Discussion by KooYS** [vercel/community#6797] [Account Issue] [answered]
### Question

I've opened a new account with this email : koo@duckzill.com

I logged in with my GitHub account, but after creating a new account and logging in with email, I mistakenly linked the wrong GitHub account and lost access to my original repository. 

It seems that the previous connection information has been deleted through a new connection. Is it impossible to recover it?

**Comment by amyegan** [comment, marked as answer]
You should be able to login to you old account using the email address instead of GitHub

**Reply by KooYS** [reply]
thanks. solved it. :)
